content/actions/hosting-your-own-runners/about-self-hosted-runners.md

```diff
@@ -94,6 +94,13 @@ The following operating systems are supported for the self-hosted runner applica
 
 {% if enterpriseServerVersions contains currentVersion %}
 
+#### Architectures
```

Comment:
we don't support x86

arm32 and arm64 are linux only.

we don't support any window-arm

Reply:
Fixed